lib/normalizeAllRuleSettings.js

```diff
@@ -17,21 +16,21 @@ function normalizeAllRuleSettings(config) {
 
 	if (!config.rules) return config;
 
-	Object.keys(config.rules).forEach((ruleName) => {
-		const rawRuleSettings = _.get(config, ['rules', ruleName]);
+	for (const ruleName of Object.keys(config.rules)) {
+		const rawRuleSettings = config.rules[ruleName];
 
```

Comment:
[suggestion] Using `for...of` is nice! 👍🏼 

In addition, how about using `Object.entries()` for more simplicity?

```suggestion
	for (const [ruleName, rawRuleSettings] of Object.entries(config.rules)) {
```